src/routers/handlers/accounts_submitted/accounts.submitted.ts

```diff
@@ -0,0 +1,61 @@
+import { BaseViewData, GenericHandler } from "../generic";
+import { Request, Response } from "express";
+import { getCompanyName, getCompanyNumber, getPaymentType, getTransactionId, getUserProfile } from "./../../../utils/session";
+import { CompanyProfile } from "@companieshouse/api-sdk-node/dist/services/company-profile";
+import { getPayment } from "./../../../utils/constants/paymentTypes";
+
+interface AccountsSubmittedViewData extends BaseViewData {
+        transactionId: string | Error,
+        companyProfile: Pick<CompanyProfile, "companyName" | "companyNumber">
+        payment: string
+        userEmail: string
+}
+
+export class AccountsSubmittedHandler extends GenericHandler{
+    constructor() {
+        super({
+            title: "Accounts Submitted",
+            backURL: null
+        });
+    }
+
+    async execute(req: Request, _response: Response
+    ): Promise<AccountsSubmittedViewData> {
+
+        super.populateViewData(req);
+
+        if (typeof req.session === "undefined"){
+            throw new Error("Session not properly set");
+        }
+
+        const companyName = getCompanyName(req.session);
+        const transactionId = getTransactionId(req.session);
+        const companyNumber = getCompanyNumber(req.session);
+        const userEmail = getUserProfile(req.session!)!.email;
+
+        const payment = getPayment(getPaymentType(req.session));
```

Comment:
Instead of using a new session value, couldn't you just pass the package type in here and then change the payment map to use the package type as the keys?

Reply:
I thought we removed the package type already from private api. Do you want me to read it?
From the package screen the type will be set in session then used in submit. That is my understanding.

Reply:
The package type is in the session is it not? There is a function to get package type there

Reply:
True I missed thanks